Update browserbench.org for Speedometer 3.0 launch

rniwa opened the pull request:
#### 194f965702debe65d73440fdb751b275f55e6284
<pre>
Update browserbench.org for Speedometer 3.0 launch
<a href="https://bugs.webkit.org/show_bug.cgi?id=269890">https://bugs.webkit.org/show_bug.cgi?id=269890</a>

Reviewed by Yusuke Suzuki.

Update the top page to link to Speedometer 3.0 instead of Speedometer 2.1.

Also address the following issues:
<a href="https://github.com/WebKit/Speedometer/issues/379">https://github.com/WebKit/Speedometer/issues/379</a>
<a href="https://github.com/WebKit/Speedometer/issues/345">https://github.com/WebKit/Speedometer/issues/345</a>

* Websites/browserbench.org/Speedometer/index.html:
* Websites/browserbench.org/Speedometer2.0/index.html:
* Websites/browserbench.org/Speedometer2.1/index.html:
* Websites/browserbench.org/Speedometer3.0/: Renamed from Websites/browserbench.org/Speedometer3.0-preview/
and updated its content to the latest.
* Websites/browserbench.org/index.html:
* Websites/browserbench.org/resources/main.css:

Canonical link: <a href="https://commits.webkit.org/275760@main">https://commits.webkit.org/275760@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/5ca064b4289296a1c531c7969bd1c69163cd474c

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [❌ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/42729 "6629 style errors") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/21750 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/45130 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/45341 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/38853 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/45035 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/25415 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/19115 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/35366 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/43302 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/18756 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/36773 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/16322 "Passed tests") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/16388 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/37834 "Passed tests") | [✅ 🛠 wpe-skia](https://ews-build.webkit.org/#/builders/52/builds/785 "Built successfully") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/20/builds/36185 "Built successfully and passed tests") | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/38901 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/38165 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/46847 "Built successfully") | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/12/builds/42356 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/17547 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/14466 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/42085 "Passed tests") | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/46/builds/19166 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/40714 "Passed tests") | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/19345 "Built successfully") | | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/35/builds/49366 "Built successfully") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/5786 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/18811 "Built successfully") | | [✅ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/25/builds/9990 "Passed tests") | 
<!--EWS-Status-Bubble-End-->

camillobruni commented on Websites/browserbench.org/index.html:
```diff
@@ -39,6 +39,15 @@
 
 <hr>
 
+<a class="benchmark" href="Speedometer3.0" tabindex="0">
```
Somehow when running locally this only seems to work with a trailing `/`, I guess that's auto-added by your server?

Constellation reviewed (approved):
r=me

rniwa committed:
Update browserbench.org for Speedometer 3.0 launch
https://bugs.webkit.org/show_bug.cgi?id=269890

Reviewed by Yusuke Suzuki.

Update the top page to link to Speedometer 3.0 instead of Speedometer 2.1.

Also address the following issues:
https://github.com/WebKit/Speedometer/issues/379
https://github.com/WebKit/Speedometer/issues/345

* Websites/browserbench.org/Speedometer/index.html:
* Websites/browserbench.org/Speedometer2.0/index.html:
* Websites/browserbench.org/Speedometer2.1/index.html:
* Websites/browserbench.org/Speedometer3.0/: Renamed from Websites/browserbench.org/Speedometer3.0-preview/
and updated its content to the latest.
* Websites/browserbench.org/index.html:
* Websites/browserbench.org/resources/main.css:

Canonical link: https://commits.webkit.org/275760@main
